### PR TITLE
Added Duplicate-Button for "datatable"-mode #149

### PR DESCRIPTION
**Pull request by Deesen**

### CODE REVIEW

**Comment by Fr3ddy7** [comment]
http://postimg.org/gallery/1wd4zx09u/

no image for this button

**Comment by Deesen** [comment]
But commit https://github.com/Jako/multiTV/pull/157/commits/cbfcfa893d031e1194b529e41e723fe5868bc5a6 is the image. Translation is missing indeed.

File: assets/tvs/multitv/js/multitv.js [hunk 1]
```diff
@@ -1013,22 +1024,24 @@
...
-            var saveTab = $('[name^="' + _this.tvid + 'tab_radio_mtv"]', _this.fieldEditForm).getValue();
-            values.fieldTab = (saveTab !== '') ? saveTab : '';
-            $.each(_this.fieldNames, function () {
-                var fieldInput = $('[name^="' + _this.tvid + this + '_mtv"][type!="hidden"]', _this.fieldEditForm);
-                values[this] = fieldInput.getValue();
-                if (fieldInput.hasClass('mtvImage')) {
-                    _this.setThumbnail(values[this], fieldInput.attr('name'), _this.fieldEditForm);
-                }
-            });
+            if(values === false) {
```
Comment (Jako) [inline assets/tvs/multitv/js/multitv.js]: I would prefer undefined over false: `(typeof values === "undefined")` . It occurs automatic if the function call gets fewer arguments than it was declared with.

File: assets/tvs/multitv/js/multitv.js [hunk 2]
```diff
@@ -673,10 +675,19 @@
...
+                            return false;
+                        }
+                        _this.duplicateRow($('.row_selected', _this.fieldTable)[0]);
+                    });
+
                     // save/append edit box
                     $('.edit,.append', _this.fieldEditForm).click(function (e) {
                         e.preventDefault();
-                        _this.saveRow($(this).hasClass('edit') ? 'edit' : 'append');
+                        _this.saveRow($(this).hasClass('edit') ? 'edit' : 'append', false);
```
Comment (Jako) [inline assets/tvs/multitv/js/multitv.js]: This has to be changed too then (remove the false).

**Comment by Deesen** [comment]
Thanks..

File: assets/tvs/multitv/js/multitv.js (hunk 2 above)
Comment (Deesen) [inline assets/tvs/multitv/js/multitv.js]: Thanks..

**Comment by Jako** [comment]
Have you checked the changes? I just read the code.

**Comment by Deesen** [comment]
Yes works for me. But for some reason the "formtab-radio" is not fully recogized when editing an existing row (radio-button not marked as active, only last manually chosen tab is remembered, not the saved one). Will investigate into this later.

**Comment by Jako** [comment]
During editing a duplicated line?

**Comment by Deesen** [comment]
No, also when adding new lines to an empty fresh TV.

**Comment by Deesen** [comment]
Happens also with MultiTV 2.0.6 of 25.01.2016 so I guess its related to some new feature in MODX like the new datepicker or whatever. Will try to find the reason.

**Comment by Jako** [comment]
Any javascript errors in the console?

**Comment by Deesen** [comment]
Unfortunately no errors.

**Comment by Deesen** [comment]
When I change

https://github.com/Jako/multiTV/blob/master/assets/tvs/multitv/js/multitv.js#L885

to 

    _this.editBoxOpen(mode, lineValue);

tabs are remembered again correctly.

**Comment by Jako** [comment]
Maybe a bug.

**Pull request merged**